scripts/pipeline-tests.yaml

```diff
@@ -0,0 +1,31 @@
+- hosts: localhost
```

Comment:
very cool - though do we want to use Ansible, rather than the likes of bash? 

Thinking of the supportability/team skills - could we have a quick chat on this?

Reply:
Sure, the choice of ansible is mainly due to the requirement of having junit/publish tests as part of the pipeline. Ansible seems to be perfect for this as it can report each task into junit XML + it can execute bash/powershell and literally everything else. In this case we're using the native get_uri module but any other can be used as well.

Reply:
> requirement of having junit/publish tests as part of the pipeline

every test framework I'm aware of can do that?

Reply:
true, but bash on it's own doesn't. So it's down to choosing a testing framework and then "Thinking of the supportability/team skills" for such framework or just use Ansible YAML.